server/datastore/mysql/statistics.go

```diff
@@ -32,6 +32,30 @@ func (ds *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Du
 		if err != nil {
 			return ctxerr.Wrap(ctx, err, "amount users")
 		}
+		amountSoftwares, err := amountSoftwaresDB(ctx, ds.writer(ctx))
+		if err != nil {
+			return ctxerr.Wrap(ctx, err, "amount Softwares")
```

Comment:
Nit: should be `amount software`, `amount host_software`, `amount software_titles`, etc. (using table names basically)